gramex/handlers/formhandler.py

```diff
@@ -53,6 +53,7 @@ def setup(cls, **kwargs):
         cls.headers = conf_kwargs.pop('headers', {})
         # Top level formats: key is special. Don't treat it as data
         cls.formats = conf_kwargs.pop('formats', {})
+        cls.task = conf_kwargs.pop('task', {})
```

Comment:
@MSanKeys963 We should not have to make any changes to FormHandler.

Reply:
Can we keep this line?
It takes care of `task: sentiment-analysis` from `gramex.yaml`.

Reply:
Why do we need `cls.task` in FormHandler? It doesn't mean anything to the FormHandler class. This attribute is just left here, and the class doesn't do anything with it. Also, if it's just a matter of popping the task key from the `conf_kwargs` dict, that can easily be done in `MLHandler` itself, before calling `super(MLHandler, ...)`

Reply:
Got it. Thank you!